src/azul/__init__.py

```diff
@@ -1566,8 +1566,15 @@ def docker_image_manifests_path(self) -> Path:
 
     allowed_v4_ips_term = 'allowed_v4_ips'
 
+    # This rate rule blocks requests when its threshold is exceeded.
+    #
     waf_rate_rule_name = 'RateRule'
 
+    # This rate rule has a higher threshold than the former, and triggers an
+    # alarm in addition to blocking requests when its threshold is exceeded.
+    #
+    waf_rate_alarm_rule_name = 'RateAlarmRule'
+
```

Comment:
I don't think this is the right place to document the semantics of these rules. Instead, that should be done in the TF code.